packages/main/test/specs/MultiComboBox.spec.js

```diff
@@ -240,6 +240,20 @@ describe("MultiComboBox general interaction", () => {
 
 			assert.strictEqual(tokens.length, 2, "2 tokens are visible");
 		});
+
+		it ("Value should be reset on ESC key", () => {
+			browser.url(`http://localhost:${PORT}/test-resources/pages/MultiComboBox.html`);
+			
+			const mCombo = $("#another-mcb");
+			const input = mCombo.shadow$("#ui5-multi-combobox-input");
+	
+			input.click();	
+			input.keys("C");
+			input.keys("Escape");
+			input.keys("Escape");
```

Comment:
why 2 times?

Reply:
The first escape closes the suggestions popover.